templates/flutter/lib/client.dart.twig

```diff
@@ -86,8 +86,10 @@ class Client {
             final Directory cookieDir = await _getCookiePath();
             cookieJar = new PersistCookieJar(dir:cookieDir.path);
             this.http.interceptors.add(CookieManager(cookieJar));
-            PackageInfo packageInfo = await PackageInfo.fromPlatform();
-            addHeader('Origin', 'appwrite-' + type + '://' + packageInfo.packageName);
+            if(Platform.isAndroid || Platform.isIOS) {
```

Comment:
@eldadfux the problem with that is PackageInfo plugin is not yet supported in Desktops in flutter, so checking only `!web` it will fail to build in Desktop, we need separate override for desktop what do we add the origin for desktop?

Reply:
The origin helps us detect the platform for future usage stats. Can we handle other desktop platforms origin manually instead?

Reply:
Yes, we sure can send platform origin header based on platform, we just need to fix what value we set for each desktop platform.

Reply:
Maybe something like `addHeader('Origin', 'appwrite-windows://');`? we won't have the package name, but we could at least detect the platform until package name will be supported.

Reply:
Will make the adjustments.

Reply:
@eldadfux  Have made the adjustments please review and let me know if there's anything else needed